index.js

```diff
@@ -135,13 +135,15 @@ module.exports.default = module.exports;
 module.exports.download = (win, url, options) => new Promise((resolve, reject) => {
 	options = Object.assign({}, options, {unregisterWhenDone: true});
 
-	registerListener(win.webContents.session, options, (err, item) => {
+	const webContents = win.webContents || win.getWebContents();
```

Comment:
`win.getWebContents` doesn't exist on Electron 7, from what I can remember, so there must be a better way to solve this.

Reply:
Apparently it's [planned to be removed in Electron 9](https://github.com/electron/electron/blob/master/docs/api/breaking-changes.md#webviewgetwebcontents). Because I do not know of a better solution to do this and no longer have time to spend on finding one I'll close the PR.